com_migratetojoomla/admin/src/View/Main/HtmlView.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Joomla\Component\MigrateToJoomla\Administrator\View\Main;
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\MVC\View\HtmlView as BaseHtmlView;
+
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_migratetojoomla
+ *
+ * @copyright   
+ * @license     GNU General Public License version 3; see LICENSE
+ */
+
+/**
+ * Main "Migrate To Joomla" Admin View
+ */
+class HtmlView extends BaseHtmlView {
```

Comment:
Make sure you keep the coding conventions of Joomla. For instance:
`class HtmlView extends BaseHtmlView {`
should be 
```
class HtmlView extends BaseHtmlView
{
```